lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -208,6 +211,18 @@ def self.is_firebasecmd_supported?(cmd)
         true
       end
 
+      def self.post_notes(app_id, release_id, release_notes)
+        payload = { releaseNotes: { releaseNotes: release_notes } }
+        if release_notes.nil? || release_notes.empty?
+          UI.message("No Release notes passed in. Skipping this step")
+          return
+        end
+        connection.post("#{PATH}#{app_id}/releases/#{release_id}/notes", payload.to_json) do |request|
```

Comment:
Two thoughts on this...
1) `PATH` is a pretty overloaded term.  I notice there's also a reference to an env variable $PATH.  I'm concerned of accidental overloading here.  Consider renaming our constant to `V1_ALPHA_APPS_PATH` to be more intention-revealing and also to disambiguate.
2) I noticed there are three (maybe more?) spots where we concatenate `#{PATH}{app_id}` -- maybe make this a helper method like
3) Consider dropping the trailing slash from the constant value and putting the slash right in the string concatenation, then it's clear where the path separator is between `PATH` and `app_id}
```
def self.v1_apps_path(app_id)
   "#{V1_ALPHA_APPS_PATH}/#{app_id}
end
```

Reply:
Thought three seems like a good idea. Will update that now to reflect this change.